Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -1103,7 +1103,11 @@ static bool SetCVarFromConsoleCommand(ICVar* cvar, AZ::ConsoleFunctorBase* conso
         {
             if constexpr (expectedCvarType != CVAR_STRING)
             {
-                cvar->Set(AZStd::to_string(value).c_str());
+                auto stringified = AZStd::to_string(value);
+                if (!stringified.empty())
+                {
+                    cvar->Set(stringified.c_str());
+                }
```

Comment:
There is no need to convert the string to a string. It is already a string.
```suggestion
                auto stringified = AZStd::to_string(value);
                if (!stringified.empty())
                {
                    cvar->Set(value.c_str());
                }
```

Reply:
this handles the case when the cvar type is actually a string, but the expected type (and therefore the T type) was not. So, although the cvar is a string, "value" is not